odxtools/expectedident.py

```diff
@@ -0,0 +1,40 @@
+# SPDX-License-Identifier: MIT
+from dataclasses import dataclass
+from typing import Any
+from xml.etree import ElementTree
+
+from .element import IdentifiableElement
+from .identvalue import IdentValue
+from .odxdoccontext import OdxDocContext
+from .odxlink import OdxLinkDatabase, OdxLinkId
+from .snrefcontext import SnRefContext
+from .utils import dataclass_fields_asdict
+
+
+@dataclass(kw_only=True)
+class ExpectedIdent(IdentifiableElement):
+    """
+    Corresponds to ADDITIONAL-AUDIENCE.
```

Comment:
`ADDITIONAL-AUDIENCE` ?

Reply:
copy and paste: best friend, worst enemy. [fixed](https://github.com/mercedes-benz/odxtools/pull/422/commits/cece9ced4375492629941f755d4db1d7a8154931), thanks.